### DISCUSSION TITLE
More granular auto-approval control.

### DISCUSSION THREAD

**Discussion by cakgok** [sct/overseerr#2824] [Feature Requests]
Is it possible to have more control on which quality profile gets auto approved? I'm perfectly fine 720p stuff getting auto-approved but not on 1080's. Especially if it is a TV show.  Currently it is divided as 4K and non-4K. I would like to option to set auto-approve based on the quality profile.

**Comment by Will695** [comment]
Just make 2 indexers and set the priority 1 with just 720p and 50 with the rest in Sonarr. Sonarr will search for 720p 1st and then if it can't find it. It will use the other indexer to search for the rest.
